src/main/resources/assets/appliedenergistics2/models/item/cable_energy_acceptor.json

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
Is the model functionally different or can you inherit the block model ?

Reply:
It has a different model. Similar to the part and block interface.